tmt/base.py

```diff
@@ -1125,6 +1148,15 @@ def _filters_conditions(self, nodes, filters, conditions):
             except fmf.utils.FilterError as error:
                 # Handle missing attributes as if filter failed
                 continue
+            # Links
+            try:
+                # Links are in OR relation
+                if links and all([not node.has_link(link_)
+                                 for link_ in links]):
+                    continue
+            except BaseException:
+                raise
+                continue
```

Comment:
Forgotten `continue`?

Reply:
The other way, forgotten 'raise' - all "filters" skip when are broken.

Reply:
I see. Previously, upon providing an incorrect relation, a proper error was shown to the user. Now, when arbitrary regexp can be used I guess it's not possible to do any kind of check, right?